pkg/clustermesh/clustermesh.go

```diff
@@ -92,6 +93,38 @@ type ClusterMesh struct {
 	nodeName string
 }
 
+type ClusterMeshUsedIds struct {
```

Comment:
```suggestion
type ClusterMeshUsedIDs struct {
```

Very minor nit, just for consistency. It also applies to sibling methods/fields which include `ID` in their name.